spring-cloud-commons/src/main/java/org/springframework/cloud/client/discovery/simple/SimpleDiscoveryProperties.java

```diff
@@ -0,0 +1,29 @@
+package org.springframework.cloud.client.discovery.simple;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Properties to hold the details of a
+ * {@link org.springframework.cloud.client.discovery.DiscoveryClient} service instances
+ * for a given service
+ *
+ * @author Biju Kunjummen
+ */
+
+@ConfigurationProperties(prefix = "spring.cloud.discovery.client.simple")
+public class SimpleDiscoveryProperties {
+	private Map<String, List<SimpleServiceInstance>> instances = new HashMap<>();
+
+	public Map<String, List<SimpleServiceInstance>> getInstances() {
+		return instances;
+	}
+
+	public void setInstances(Map<String, List<SimpleServiceInstance>> instances) {
```

Comment:
Yes, thought about that @spencergibb, the only reason was `DefaultServiceInstance` has all final fields and that would not work with mapping via `@ConfigurationProperties` right